server/plugin/plugin.go

```diff
@@ -376,13 +379,58 @@ func (p *Plugin) CreateBotDMPost(userID, message, postType string) {
 	}
 }
 
+func (p *Plugin) CheckIfDuplicateDailySummary(userName, text string) (bool, error) {
+	previousSummary, err := p.GetDailySummaryText(userName)
+	if err != nil {
+		return false, err
+	}
+	if previousSummary == text {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+func (p *Plugin) StoreDailySummaryText(userName, summaryText string) error {
+	if err := p.API.KVSet(userName+dailySummary, []byte(summaryText)); err != nil {
```

Comment:
Does it make sense to store this using the Mattermost user ID instead of the github username?

Reply:
yes we can